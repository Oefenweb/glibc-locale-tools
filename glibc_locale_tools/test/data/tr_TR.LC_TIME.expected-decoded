comment_char %
escape_char  /
abday   "Paz";"Pzt";"Sal";/
        "Çrş";"Prş";"Cum";"Cts"
day     "Pazar";/
	"Pazartesi";/
	"Salı";/
	"Çarşamba";/
	"Perşembe";/
	"Cuma";/
	"Cumartesi"
abmon   "Oca";"Şub";/
	"Mar";"Nis";/
        "May";"Haz";/
        "Tem";"Ağu";/
        "Eyl";"Eki";/
	"Kas";"Ara"
mon     "Ocak";/
	"Şubat";/
	"Mart";/
	"Nisan";/
	"Mayıs";/
        "Haziran";/
        "Temmuz";/
	"Ağustos";/
        "Eylül";/
        "Ekim";/
	"Kasım";/
	"Aralık"
d_t_fmt "%a %d %b /
%Y %T %Z"
d_fmt   "%d-%m-%Y"
t_fmt   "%T"
am_pm   "ÖÖ";"ÖS"
t_fmt_ampm "%I:%M:%S/
 %p"
date_fmt       "%a %b %e/
 %H:%M:%S /
%Z %Y"
%  FIXME: found in CLDR
first_weekday 2
