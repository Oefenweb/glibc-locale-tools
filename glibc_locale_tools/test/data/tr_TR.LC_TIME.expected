comment_char %
escape_char  /
abday   "<U0050><U0061><U007A>";"<U0050><U007A><U0074>";"<U0053><U0061><U006C>";/
        "<U00C7><U0072><U015F>";"<U0050><U0072><U015F>";"<U0043><U0075><U006D>";"<U0043><U0074><U0073>"
day     "<U0050><U0061><U007A><U0061><U0072>";/
	"<U0050><U0061><U007A><U0061><U0072><U0074><U0065><U0073><U0069>";/
	"<U0053><U0061><U006C><U0131>";/
	"<U00C7><U0061><U0072><U015F><U0061><U006D><U0062><U0061>";/
	"<U0050><U0065><U0072><U015F><U0065><U006D><U0062><U0065>";/
	"<U0043><U0075><U006D><U0061>";/
	"<U0043><U0075><U006D><U0061><U0072><U0074><U0065><U0073><U0069>"
abmon   "<U004F><U0063><U0061>";"<U015E><U0075><U0062>";/
	"<U004D><U0061><U0072>";"<U004E><U0069><U0073>";/
        "<U004D><U0061><U0079>";"<U0048><U0061><U007A>";/
        "<U0054><U0065><U006D>";"<U0041><U011F><U0075>";/
        "<U0045><U0079><U006C>";"<U0045><U006B><U0069>";/
	"<U004B><U0061><U0073>";"<U0041><U0072><U0061>"
mon     "<U004F><U0063><U0061><U006B>";/
	"<U015E><U0075><U0062><U0061><U0074>";/
	"<U004D><U0061><U0072><U0074>";/
	"<U004E><U0069><U0073><U0061><U006E>";/
	"<U004D><U0061><U0079><U0131><U0073>";/
        "<U0048><U0061><U007A><U0069><U0072><U0061><U006E>";/
        "<U0054><U0065><U006D><U006D><U0075><U007A>";/
	"<U0041><U011F><U0075><U0073><U0074><U006F><U0073>";/
        "<U0045><U0079><U006C><U00FC><U006C>";/
        "<U0045><U006B><U0069><U006D>";/
	"<U004B><U0061><U0073><U0131><U006D>";/
	"<U0041><U0072><U0061><U006C><U0131><U006B>"
d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020>/
<U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
d_fmt   "<U0025><U0064><U002D><U0025><U006D><U002D><U0025><U0059>"
t_fmt   "<U0025><U0054>"
am_pm   "<U00D6><U00D6>";"<U00D6><U0053>"
t_fmt_ampm "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053>/
<U0020><U0025><U0070>"
date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
<U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
<U0025><U005A><U0020><U0025><U0059>"
%  FIXME: found in CLDR
first_weekday 2
